tests/5.0/requires/test_requires_reverse_offload.c

```diff
@@ -41,6 +42,8 @@ int main()
     }
 
     OMPVV_WARNING_IF(device_num <= 0, "Cannot properly properly test reverse offload if no devices are available");
+    OMPVV_TEST_AND_SET_SHARED_ENVIRONMENT(is_shared_env);
+    OMPVV_WARNING_IF(is_shared_env != 0, "[WARNING] May not be able to detect errors if the target system supports shared memory.")
```

Comment:
The same applies for the C testcase as to the Fortran testcase: You need an the reverse-offload/ancestor:1 target construct needs to be inside a target region to be effective.

Reply:
Updated.